### ISSUE TITLE
Get mAP after training (Inference)

### ISSUE THREAD

**Issue by MahBadran93** [tensorflow/models#10872]
I am trying to get the model performance after training. In most documentation and articles I read, they only do a validation step using the model_main_tf2.py file during training. But this doesn't work without having the training mode active and If I run it without the training mode it will stuck at an evaluation loop (waiting for a new checkpoint) and I get the below message:

`INFO:tensorflow:Waiting for new checkpoint at /content/drive
Waiting for new checkpoint at /content/drive`

My main question is how to do find **mAP**  values after training (during inference)?  I have exported a .db model.

Pretrained model used: **faster_rcnn_resnet101_v1_1024x1024_coco17_tpu-8**
Trained on Colab

**Comment by laxmareddyp** [comment]
Hi @MahBadran93,

If you have any validation data with ground truth available, you can use the `model_main_tf2.py` in evaluation mode and get the eval  metrics and also could you please share the reproducible colab code to check from our side.

Thanks.
